README.md

```diff
@@ -235,25 +235,10 @@ Please be aware that Giraffe strictly aims to remain as light as possible while
 
 ### Code conventions
 
-When making changes please use existing code as a guideline for coding style and documentation. For example add spaces when creating tuples (`(a,b)` --> `(a, b)`), annotating variable types (`str:string` --> `str : string`) or other language constructs.
+We use [Fantomas](https://fsprojects.github.io/fantomas/docs/index.html) to ensure consistent code formatting across the project. Before committing, format your code by running the following command:
```

Comment:
I did not do any attempt to keep the rules. They were implemented inconsistent anyway and I feel like the default from the style guide suffice here.